README.md

```diff
@@ -50,13 +50,15 @@ for Godot by following these steps:
 
 You can use the following settings to configure Godot Tools:
 
-- `editor_path` - The absolute path to the Godot editor executable.
+- `editor_path` - The absolute path to the Godot editor executable. _Under Mac OS, this is the executable inside of Godot.app._
 - `gdscript_lsp_server_port` - The WebSocket server port of the GDScript language server.
 - `check_status` - Check the GDScript language server connection status.
 
-#### Debugger
+#### GDScript Debugger
 
-To configure the debugger:
+The debugger is for GDScript projects. To debug C# projects, use [Mono Debug](https://godotsharp.net/how-to/debug-your-mono-godot-project/).
```

Comment:
We don't know who administrates godotsharp.net nor how up to date its content is. It would better to make sure the relevant config is available in Godot's own official documentation, and link that.

Reply:
And this howto does seem quite outdated to me. It doesn't mention the official https://github.com/godotengine/godot-csharp-vscode in any way.